packages/config/config/devices/0x0258/Motion Sensor 3

```diff
@@ -0,0 +1,174 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "Motion Sensor 3",
+	"description": "PIR Motion Sensor 3",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x6082"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Sensitivity Level",
+			"description": "Defines the sensitivity of PIR sensor.",
```

Comment:
What's high, what's low?

Reply:
I just copied this part from the other Neo Coolcam motion sensors. 
In the manual 8 is highest sensitivy, and 255 means lowest sensitivity.

Reply:
I've added the information into the description.